spec/models/manageiq/providers/ovirt/infra_manager/refresher/ovirt_refresher_spec_common.rb

```diff
@@ -6,26 +6,23 @@ module OvirtRefresherSpecCommon
   def init_defaults(hostname: "pluto-vdsg.eng.lab.tlv.redhat.com",
                     ipaddress: "10.35.19.13",
                     port: 443,
-                    external_network_provider_mock: 'external_network_providers')
+                    username: "admin@internal",
+                    password: "pass123")
     _guid, _server, @zone = EvmSpecHelper.create_guid_miq_server_zone
-    create_ems(:hostname => hostname, :ipaddress => ipaddress, :port => port, :zone => @zone, :external_network_provider_mock => external_network_provider_mock)
-    init_inventory_wrapper_class
+    create_ems(:hostname => hostname, :ipaddress => ipaddress, :port => port, :username => username, :password => password, :zone => @zone)
   end
 
   def create_ems(hostname: "pluto-vdsg.eng.lab.tlv.redhat.com",
                  ipaddress: "10.35.19.13",
                  port: 443,
-                 zone: @zone,
-                 external_network_provider_mock: 'external_network_providers')
+                 username: "admin@internal",
+                 password: "pass123",
+                 zone: @zone)
     @ems = FactoryBot.create(:ems_ovirt, :zone => zone, :hostname => hostname, :ipaddress => ipaddress,
                              :port => port)
-    @ems.update_authentication(:default => {:userid => "admin@internal", :password => "pass123"})
+    @ems.update_authentication(:default => {:userid => username, :password => password})
     @ems.default_endpoint.verify_ssl = OpenSSL::SSL::VERIFY_NONE
     @ems.default_endpoint.path       = "/ovirt-engine/api"
-
-    @ovirt_service = ManageIQ::Providers::Ovirt::InfraManager::OvirtServices::V4
-    allow_any_instance_of(@ovirt_service)
-      .to receive(:collect_external_network_providers).and_return(load_response_mock_for(external_network_provider_mock))
```

Comment:
This is a straightforward API call which was captured in the response YAML and didn't need to be stubbed out